openslides_backend/action/actions/organization/initial_import.py

```diff
@@ -84,6 +87,24 @@ def check_empty_datastore(self) -> None:
         ):
             raise ActionException("Datastore is not empty.")
 
+    def translate_organization_and_theme(self, data: Dict[str, Any]) -> None:
+        if "organization" in data and "1" in data["organization"]:
```

Comment:
call this method after the checker so you can be sure that the organization and the id exist and you can skip this check

Reply:
I have moved the translate method behind the checker and removed this check.